File: csharp/DiagramPrinter/DiagramPagesReport.cs
namespace DiagramPrinter;

public class DiagramPagesReport
{
    private List<DiagramReportPage> pages;

    public void AddPage(DiagramReportPage page)
    {
        this.pages.Add(page);
    }

    public bool IsComplete()
    {
        return pages.Count != 0;
    }
}

File: java/src/main/java/sammancoaching/DiagramPagesReport.java
package sammancoaching;

import java.util.ArrayList;
import java.util.List;

public class DiagramPagesReport {
    private List<DiagramReportPage> pages;

    public DiagramPagesReport() {
        this.pages = new ArrayList<>();
    }

    public void addPage(DiagramReportPage page) {
        this.pages.add(page);
    }

    public boolean isComplete() {
        return !pages.isEmpty();
    }
}